be/src/runtime/routine_load/data_consumer_group.cpp

```diff
@@ -104,6 +112,15 @@ Status KafkaDataConsumerGroup::start_all(StreamLoadContext* ctx) {
         }
     }
 
+    char errstr[512];
```

Comment:
> Since the `serdes` is only needed for `avro`, how about moving this to line 231?

231 line is in a while loop. We only need the serdes object to be initialized once.